torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -1,3 +1,4 @@
+#include <ATen/CUDAGeneratorImpl.h>
```

Comment:
why do we need this #include ?

Reply:
You're right, we don't need it.  I instinctively included it because I mention PhiloxCudaState in the file, but all we're doing in the file is building strings.